.solhint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "solhint:recommended",
+  "rules": {
+    "code-complexity": ["error", 8],
+    "compiler-version": ["error", ">=0.8.17"],
```

Comment:
We should use >0.8.21 because of https://soliditylang.org/blog/2023/07/19/missing-side-effects-on-selector-access-bug/